testing/features/input.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Input
+  As a non-technical person at Citizens Advice
```

Comment:
Lets alter this to be more specificatory to the component proper (We would usually do this style for the regular features on public-website-cucumber).

Take a look at the other components for examples.